### ISSUE TITLE
Voice over bugs

### ISSUE THREAD

**Issue by Gargron** [mastodon/mastodon-ios#309]
Analysis provided by @marcozehe [here](https://toot.cafe/@marcozehe/107352653997388458)

- [ ] The text in the header says „mastodon.text“, claims is a button, but double-tapping only scrolls to the top
- [ ] Posts behind CWs aren‘t indicated. I get the content without warning.
- [ ] Time stamps spoken get terribly wrong once I start scrolling the screen
- [ ] Notification screen has multiple stops per item
- [ ] Posts in main screen have no custom actions to reply etc
- [ ] Poll options and their current values in my original post aren‘t accessible at all.
- [ ] The theme of messed up time stamps on posts continues in detail/conversation screens.
- [ ] Reply button claims there are 0 reblogs if a post has no replies, boosts and favorites. Even if it has, the Reply button still talks about reblogs instead of replies.

**Comment by Bracciata** [comment]
In the current version, the header's text says " logo button button". Do you have an opinion on what it should be upgraded to? Potentially "Scroll to top button"?
